src/button/button.module.scss

```diff
@@ -1,7 +1,4 @@
-.buttonColor {
```

Comment:
I don't know if you want to remove this or it was en error. Maybe the button color was changing because of this